Kernel/Graphics/VirtIOGPU/GPU3DDevice.h

```diff
@@ -130,13 +143,15 @@ class VirtIOGPU3DDevice : public CharacterDevice {
     virtual ErrorOr<void> ioctl(OpenFileDescription&, unsigned request, Userspace<void*> arg) override;
     virtual void detach(OpenFileDescription&) override;
 
+    using ContextsList = IntrusiveListRelaxedConst<&PerContextState::m_list_node>;
```

Comment:
Just a nitpick: "ContextList" seems more appropriate, just as we usually write things like "ProcessList" instead of "ProcessesList".